### DISCUSSION TITLE
Add Row Greyed Out

### DISCUSSION THREAD

**Discussion by sheissportsnetwork** [AntlerVC/firetable#1072] [Support / Q&A] [answered]
I have two collections with identical permissions, but I can only add new rows in one of them. For the other, it's greyed out.

Here's my working Persons collection:
![Screenshot 2023-01-09 at 11 09 19 AM](https://user-images.githubusercontent.com/114926906/211354196-85fb387d-91ef-41c7-820e-dfadf30988a2.png)

Access controls for Persons collection:
match /Persons/{docId} {
  allow read, write: if hasAnyRole(["ADMIN","EDITOR"]);
}

And here's my Teams collection which won't allow anyone to add a row:
![Screenshot 2023-01-09 at 11 09 12 AM](https://user-images.githubusercontent.com/114926906/211354365-c010bc36-e223-4be5-bc9b-9c8022839078.png)

Access controls for Teams collection:
match /Teams/{docId} {
  allow read, write: if hasAnyRole(["ADMIN","EDITOR"]);
}

**Comment by shamsmosowi** [comment, marked as answer]
hi @sheissportsnetwork , is the Teams table a collection group type? if so, we currently don't support adding rows to it since we don't have a way if specifying the document reference path

**Reply by sheissportsnetwork** [reply]
Thanks for the fast reply. It's the same type as the Persons collection, both just collections, not collection groups.

**Reply by BeeBombshell** [reply]
Hey @sheissportsnetwork, is this still an issue?

**Reply by sheissportsnetwork** [reply]
We can use it now. While the collection was just a collection, one of the indexes in Firestore was set as a group. Deleted that and it works now 👍
